apps/postgres-new/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@ai-sdk/openai": "^0.0.21",
     "@dagrejs/dagre": "^1.1.2",
-    "@electric-sql/pglite": "^0.2.8",
+    "@electric-sql/pglite": "0.2.8",
```

Comment:
Intentional?

Reply:
Oops, it was to ensure it works for older databases, will fix!